frontends/PyCDE/test/module_naming.py

```diff
@@ -42,14 +42,12 @@ def build(self, top):
     UnParameterized(x=c1)
 
 
-# CHECK: hw.module @Module
```

Comment:
I think this test was wrong -- it was testing that two modules for the same parameterization got generated twice.

Reply:
that is what it tested, and that was the previous behavior... that was a casualty of the previous, simplistic approach to avoid renaming anything for modules without parameters. it should be removed now for sure.